Chef.PowerShell.Wrapper.Core/Chef.PowerShell.Wrapper.Core.cpp

```diff
@@ -3,12 +3,27 @@
 
 using namespace System;
 
-const wchar_t* ExecuteScript(const char* powershellScript, int timeout)
+// This is the entry point for the DLL. It is called from ruby with the powershell script to execute.
+// Note that this is for "PowerShell Core" (6.0 and later) and not "PowerShell" (5.1 and earlier).
+// You likely want to make similar changes to the Chef.PowerShell.Wrapper.cpp file.
```

Comment:
this last comment line should probably be removed since it was done.

Reply:
This is for anyone trying to change this in the future... otherwise you'll end up with a fix that breaks in a few test cases and spend hours questioning your sanity.